libkbfs/bcache.go

```diff
@@ -238,6 +242,8 @@ func (b *BlockCacheStandard) PutWithPrefetch(
 		defer func() {
 			if madeRoom {
 				b.cleanTransient.Add(ptr.ID, blockContainer{block, hasPrefetched})
+			} else {
+				err = cachePutCacheFullError{ptr}
```

Comment:
Thanks!  It definitely should be.  Looks like a pre-existing bug that was made much more likely by `PutWithPrefetch`.